server/views/pages/approve/prisonerSummaryDetails.njk

```diff
@@ -0,0 +1,26 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{% macro prisonerSummaryDetails(licence) %}
+    {{ govukSummaryList({
+        attributes: { id: 'prisoner-summary-details' },
+        classes: 'govuk-summary-list--no-border',
+        rows: [
+            {
+                key: { text: "Name" },
+                value: { text: licence.appointmentPerson }
```

Comment:
should this be the prisoner name?

Reply:
No we are getting the value in appointmentPerson field